src/lib/Components/Component.php

```diff
@@ -5,4 +5,6 @@
 interface Component extends \JsonSerializable
 {
     public function __toString();
```

Comment:
I think you can remove `__toString` from this interface and remove `__toString` implementation in components implementing this interface